dev/io.openliberty.http.monitor_fat/fat/src/io/openliberty/http/monitor/fat/BaseTestClass.java

```diff
@@ -325,15 +325,15 @@ protected boolean validateMpTelemetryHttp(String appName, String vendorMetricsOu
             countMatchString = "http_server_request_duration_seconds_count\\{http_request_method=\""
                                + requestMethod
                                + "\",http_response_status_code=\"" + responseStatus
-                               + "\",http_route=\"" + route
+                               + ((route != null) ? ("\",http_route=\"" + route) : "")
```

Comment:
Line 328 is different than line 269.  Is this intentional?

Reply:
Yup, `mpMetrics` must have the same amount of tags for all metrics of the same name. Even an empty attribute/label would need to be present as other http metrics will have HTTP routes present.